cmd/eval-dev-quality/cmd/evaluate_test.go

```diff
@@ -676,6 +679,249 @@ func TestEvaluateExecute(t *testing.T) {
 		})
 	})
 
+	t.Run("Runtime", func(t *testing.T) {
+		// Skip containerized runtime tests if special cases.
+		if osutil.IsWindows() {
+			t.Skip("Docker runtime not supported on Windows")
+		} else if _, err := exec.LookPath("docker"); err != nil {
+			t.Skip("Docker runtime not found")
+		}
+
+		// TODO: Make the docker runtime run on both local and CI
+		if os.Getenv("GITHUB_ACTIONS") != "true" {
+			t.Skip("We are currently only supporting the Github CI")
+		}
+
+		// Prerequisite
```

Comment:
docstring that states we are are pulling the image that was just built for this branch to avoid building it again